qlty-cli/src/commands/coverage/publish.rs

```diff
@@ -83,13 +83,28 @@ pub struct Publish {
 
     #[arg(long, hide = true)]
     pub skip_missing_files: bool,
+
+    #[arg(long)]
+    /// The total number of parts that qlty cloud should expect. Each call to qlty publish will upload one part.
+    /// (The total parts count is per coverage tag e.g. if you have 2 tags each with 3 parts, you should set this to 3)
+    pub total_parts_count: Option<u32>,
```

Comment:
This is a verbose description for the arg. How does this look in the `--help` output?

Reply:
Lots of output here already ...

<img width="971" alt="image" src="https://github.com/user-attachments/assets/0b30b4a0-6cdf-4822-86f5-80cf8e7e928c" />

Reply:
Hmm OK. Let's go with this for now and I'll do an editing pass someday.

Basically we need man pages